src/shared/loadPseudoData.c

```diff
@@ -0,0 +1,29 @@
+#include "loadPseudoData.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+
+int loadData(const char *input_file, const char *output_file) {
```

Comment:
We use snake case for function names and lower camel case for parameters #3
`loadData` -> `load_data`
`input_file` -> `inputFile`
`output_file` -> `outputFile`

This function also has an int return value. But it never actually returns an int??